cirq/google/sim/xmon_simulator.py

```diff
@@ -166,6 +167,35 @@ def __init__(self, options: XmonOptions = None) -> None:
         """
         self.options = options or XmonOptions()
 
+    def wavefunction(self, state, decimals=2):
```

Comment:
This is kind of a strange place to put the method since you use the simulator to produce the result and then use it to print it.  I think it would be better if this method was a method on the XmonSimulateTrialResult and also on the XmonStepResult, so that you don't have to pass in the result.  I'd make this a global function, as it is pretty useful even if you are just given a state, and then delegate to it in those two classes.  For the global function, it would also help if the name were more explicit, i.e. something like pretty_wavefunction or pretty_state.